google/ggrc-core
# GGRC-6444 Refactor external resource service

# Issue description

Refactored resource for external models. Created ExternalResource for Control model.

# Steps to test the changes

1. Create control using external app user. Should be created.
2. Create control using internal user. Should be forbidden.

# Solution description

Created resource class for external models, that override POST, PUT, DELETE method to prevent model creation/modification for internal users.

# Sanity checklist




# PR Review checklist
